README.md

```diff
@@ -93,3 +93,4 @@ specific language governing permissions and limitations under the License.
 [slack-badge]: http://slack.linkerd.io/badge.svg
 [slack]: http://slack.linkerd.io
 [zipkin]: https://github.com/openzipkin/zipkin
+[releases]: ../../releases
```

Comment:
i would just hardcode this as `https://github.com/linkerd/linkerd/releases`, as it's something that shouldn't vary from branch-to-branch.

Reply:
Alright will do, thanks